flang/lib/Lower/IO.cpp

```diff
@@ -270,18 +270,22 @@ getNamelistGroup(Fortran::lower::AbstractConverter &converter,
       groupIsLocal = true;
       continue;
     }
-    std::string mangleName = converter.mangleName(s) + ".desc";
-    if (builder.getNamedGlobal(mangleName))
-      continue;
-    const auto expr = Fortran::evaluate::AsGenericExpr(s);
-    fir::BoxType boxTy =
-        fir::BoxType::get(fir::PointerType::get(converter.genType(s)));
-    auto descFunc = [&](fir::FirOpBuilder &b) {
-      auto box =
-          Fortran::lower::genInitialDataTarget(converter, loc, boxTy, *expr);
-      b.create<fir::HasValueOp>(loc, box);
-    };
-    builder.createGlobalConstant(loc, boxTy, mangleName, descFunc, linkOnce);
+    // We know we have a global item.  It it's not a pointer or allocatable,
+    // create a pointer to it.
```

Comment:
`a static pointer` will be even more descriptive.

Reply:
I'll change it!